lms/static/js/commerce/views/receipt_view.js

```diff
@@ -164,7 +164,7 @@ var edx = edx || {};
          * @return {object} JQuery Promise.
          */
         getCourseData: function (courseId) {
-            var courseDetailUrl = '/api/course_structure/v0/courses/%s/';
+            var courseDetailUrl = '/api/courses/v1/courses/%s/';
```

Comment:
It appears we are submitting the API request as an anonymous user, and we are receiving a valid response -- is there a reason why we need to specify the user for this particular call?  I can include that information if necessary, but it seems like it's working?